packages/bisect_ppx/bisect_ppx.0.2.2/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+name: "bisect_ppx"
+version: "0.2.2"
+maintainer: "Leonid Rozenberg <leonidr@gmail.com>"
+authors: "Leonid Rozenberg <leonidr@gmail.com>"
+homepage: "https://github.com/rleonid/bisect_ppx"
+dev-repo: "https://github.com/rleonid/bisect_ppx.git"
+bug-reports: "https://github.com/rleonid/bisect_ppx/issues"
+build: [
+  ["sh" "configure"]
+  [make "all"]
+  [make "install"]
```

Comment:
@samoht Here: [Opam Manual](https://opam.ocaml.org/doc/Manual.html#opam)

> This field contains typically just [make "install"]. It is recommended, though, to prefer the usage of a static or generated package-name.install file and omit the install: field.

Although, you're right that there is no conflict between the two.